lib/online_kitchen/labmanager4.rb

```diff
@@ -124,12 +132,47 @@ def equip_machine_ip(name)
         false
       end
 
+      def get_config(opts = {})
+        # if opts.key?(:cluster)
+        #   OnlineKitchen.logger.debug("\e[1;31m LMNAME PRESENT #{opts[:cluster]}  \e[0m")
+        # else
+        #   OnlineKitchen.logger.debug("\e[1;31m LMNAME NOT PRESENT  \e[0m")
+        # end
```

Comment:
Please consider removing this unused block of code, thank you

Reply:
done, even more lines removed